### PR TITLE
Yarkin/config optimization

### LINKED ISSUE #5: Move flask_proxy from EOSEVM repo to the this repo
https://github.com/eosnetworkfoundation/eos-evm/issues/397

### PR DESCRIPTION
**Pull request by yarkinwho**
1 Fix of some bugs.
2 Frontend UI and Style
3 Optimize config.

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.7'
```
Comment (elmato) [inline docker-compose/docker-compose.yml]: do we need to rebase from 3.8 ? or merge master first?

File: docker-compose/envs/common-blockscout.env [hunk 2]
```diff
@@ -1,11 +1,19 @@
 # DOCKER_TAG=
 SECRET_KEY_BASE=RMgI4C1HSkxsEjdhtGMfwAHfyT6CKWXOgzCboJflfSm4jeAlic52io05KB6mqzc5
+<<<<<<< HEAD
+ETHEREUM_JSONRPC_VARIANT=evm
+=======
 ETHEREUM_JSONRPC_VARIANT=erigon
+>>>>>>> origin/evm
```
Comment (elmato) [inline docker-compose/envs/common-blockscout.env]: fix this

File: docker-compose/envs/common-blockscout.env [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
 ETHEREUM_JSONRPC_HTTP_URL=http://host.docker.internal:5000
 DATABASE_URL=postgresql://postgres:@host.docker.internal:7432/blockscout?ssl=false
 ETHEREUM_JSONRPC_TRACE_URL=http://host.docker.internal:5000
 NETWORK=
+<<<<<<< HEAD
+SUBNETWORK=TrustLocal chain
+=======
 SUBNETWORK=TrustEVM chain
+>>>>>>> origin/evm
```
Comment (elmato) [inline docker-compose/envs/common-blockscout.env]: same as above

File: docker-compose/envs/common-blockscout.env (hunk 2 above)
Comment (yarkinwho) [reply docker-compose/envs/common-blockscout.env:7]: I added the evm config. 
evm is slightly different from erigon

**Review by elmato — APPROVED** [review]

**Pull request merged**